app/views/pages/home.html.slim

```diff
@@ -96,6 +96,16 @@
                   .description.right.right-side
                     h3 Sam Lau
                     = render partial: "shared/tas/sam"
+            .ta
```

Comment:
this will probably conflict with #126 so whoever gets merged second will have to fix theirs

cc @DrakeW